switch.sh

```diff
@@ -81,6 +81,7 @@ switch_on_sync_done() {
 	echo 'Bitcoind has been switched across to neutrino'
 	touch /statuses/node-status-bitcoind-ready
 	sed -Ei 's|(bitcoin.node)=neutrino|\1=bitcoind|g' /lnd/lnd.conf
+	sed -i "s/feeurl=.*//g;" /lnd/lnd.conf
```

Comment:
We should probably use something more explicit here just so it doesn't match a potential future `testnetfeeurl=` arg.

```suggestion
	sed -i "s/^feeurl=.*//g;" /lnd/lnd.conf
```